doc/release-process.md

```diff
@@ -2,6 +2,19 @@ Release Process
 ====================
 Meta: There should always be a single release engineer to disambiguate responsibility.
 
+## Pre-release
+
+Check all of the following:
+
+- All dependencies have been updated as appropriate:
+  - BDB
+  - Boost
+  - ccache
+  - libgmp
+  - libsodium
+  - miniupnpc
+  - OpenSSL
```

Comment:
I checked for completeness of this list [here](https://github.com/zcash/zcash/blob/master/depends/packages/packages.mk). Add a check for changes to upstream libsnark.